scm/src/CMakeLists.txt

```diff
@@ -159,76 +159,99 @@ else (APPLE)
   message (FATAL_ERROR "Unsupported platform, only Linux and MacOSX are supported at this time.")
 endif()
 
+# Build Options
```

Comment:
@climbfuji Would you take a quick look at these changes in CMakeLists.txt to see if they make sense? I basically copied the flag setting that you pointed to in ufs-weather-model/cmake/Intel.cmake and gnu.cmake and added the bit that sets CMAKE_Fortran_FLAGS_PHYSICS/DYNAMICS from ufs-weather-model/FV3/ccpp/CMakeLists.txt. I didn't attempt to do anything with the INSTALL command.